### PR TITLE
change serving country certificates

### PR DESCRIPTION
**Pull request by Nil20**
Country config PR: https://github.com/opencrvs/opencrvs-countryconfig/pull/250

We can token protect the farajaland `certificate` routes like in the farajaland PR which seems like a much simpler approach than creating presigned url like we discussed [here](https://github.com/opencrvs/opencrvs-core/issues/7156#issuecomment-2259938605).

### CODE REVIEW

**Comment by rikukissa** [comment]
Indeed, seems like a better approach 👍

**Comment by rikukissa** [comment]
@Zangetsu101 could you also review this please?

**Comment by Nil20** [comment]
We can update the changelog in this [PR](https://github.com/opencrvs/opencrvs-core/pull/7352)

File: packages/client/src/components/interface/Navigation.tsx [hunk 1]
```diff
@@ -687,62 +677,6 @@ const NavigationView = (props: IFullProps) => {
...
-                            id={`navigation_${WORKQUEUE_TABS.systems}`}
-                            label={intl.formatMessage(
-                              navigationMessages[WORKQUEUE_TABS.systems]
-                            )}
-                            onClick={goToSystemViewAction}
-                            isSelected={
-                              enableMenuSelection &&
-                              activeMenuItem === WORKQUEUE_TABS.systems
-                            }
-                          />
```
Comment (Zangetsu101) [inline packages/client/src/components/interface/Navigation.tsx:740]: We don't want to remove the `integrations` nav item no?

File: packages/client/src/offline/reducer.ts [hunk 2]
```diff
@@ -140,47 +139,19 @@ async function saveOfflineData(offlineData: IOfflineData) {
   return storage.setItem('offline', JSON.stringify(offlineData))
 }
 
-export type CertificatePayload = Awaited<ReturnType<typeof loadCertificate>>
+export type CertificatePayload = Awaited<
+  Promise<{
+    svgCode: string
+    event: Event
+  }>
+>
```
Comment (Zangetsu101) [inline packages/client/src/offline/reducer.ts:147]:
```suggestion
export type CertificatePayload = {
  svgCode: string
  event: Event
}
```

File: packages/client/src/offline/reducer.ts [hunk 3]
```diff
@@ -140,47 +139,19 @@ async function saveOfflineData(offlineData: IOfflineData) {
...
-  savedCertificates: IOfflineData['templates']['certificates'],
-  fetchedCertificates: ICertificateTemplateData[]
-) {
-  return await Promise.all(
-    fetchedCertificates.map((cert) =>
-      loadCertificate(savedCertificates?.[cert.event], cert)
-    )
-  )
+function loadCertificates(fetchedCertificates: ICertificateTemplateData[]) {
+  return fetchedCertificates
```
Comment (Zangetsu101) [inline packages/client/src/offline/reducer.ts:154]: Do we even need these two? They are basically identity functions :thinking:

File: packages/client/src/offline/reducer.ts
Comment (Zangetsu101) [inline packages/client/src/offline/reducer.ts:1]: I think we can simplify the reducer quite a bit now

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

**Review by Zangetsu101 — APPROVED** [review]

**Comment by Nil20** [comment]
> Indeed, seems like a better approach for now 👍 Could you please confirm the certificates also still work offline?

Yep. It does.

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**